Gemfile

```diff
@@ -58,4 +58,8 @@ end
 group :production do
   gem 'unicorn'
   gem 'rails_12factor'
+
+group :development do
```

Comment:
There's another `development` group in the Gemfile, around line 26. Want to just chuck the dawnscanner gem in there for cleanliness' sake?